blogposts/2023-02-10-adding-snapshot-testing-to-rnw-apps.md

```diff
@@ -0,0 +1,93 @@
+---
+post_title: "Adding Jest Snapshot Testing to React Native Windows Apps"
+author1: chiaramooney
+post_slug: 2023-02-10-adding-snapshot-testing-to-rnw-apps
+post_date: 2023-02-10 12:00:00
```

Comment:
headsup: I think that we will need to get either Steven or Ryan to also give their ✅, I'd propose a target publication date for mid-next week (14/15)

Reply:
Will do!